postgis/postgis.sql.in

```diff
@@ -1099,6 +1099,13 @@ CREATE OR REPLACE FUNCTION postgis_hasbbox(geometry)
 	AS 'MODULE_PATHNAME', 'LWGEOM_hasBBOX'
 	LANGUAGE 'c' IMMUTABLE STRICT _PARALLEL;
 
+-- Availability: 2.5.0
+CREATE OR REPLACE FUNCTION postgis_optimize_geometry(g geometry, num_significant_digits int)
```

Comment:
can a default of EPSILON_SQLMM = 1e-8 be set here?

Reply:
I don't think so. The link between the epsilon value and the number of significant digits would depend on the magnitude of the coordinates involved.